manifests/m/Microsoft/PowerShell/Preview/7.4.0.4/Microsoft.PowerShell.Preview.installer.yaml

```diff
@@ -0,0 +1,46 @@
+# Created with YamlCreate.ps1 v2.2.8 $debug=NVS1.CRLF.7-3-5.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Microsoft.PowerShell.Preview
+PackageVersion: 7.4.0.4
+MinimumOSVersion: 10.0.17763.0
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Commands:
+- pwsh
+ReleaseDate: 2023-06-29
+Installers:
+- Architecture: x64
+  InstallerType: wix
```

Comment:
```suggestion
  InstallerType: msi
```

Reply:
The MSI is made using WiX Toolset, so `InstallerType: wix` is more accurate in this case. When it comes to passing the silent switches, winget handles both the same way. See

https://github.com/microsoft/winget-cli/blob/e4b9205cff88701b2ad1078dd8ed311ac716b8b6/src/AppInstallerCommonCore/Manifest/ManifestCommon.cpp#L768-L776

Reply:
so what's the issue with this PR?